libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1160,3 +1167,50 @@ bool AP_AHRS_DCM::pre_arm_check(bool requires_position, char *failure_msg, uint8
     }
     return true;
 }
+
+/*
+  relative-origin functions for fallback in AP_InertialNav
+*/
+bool AP_AHRS_DCM::get_origin_fallback(Location &ret) const
```

Comment:
Why isn't this just `get_origin`?

Reply:
because I wanted this to have minimal impact on the rest of the vehicle - I don't want get_origin() to start returning true when we have home but don't yet have origin. That might be OK, but it is a much more extensive change in behaviour.
This PR is designed to only have a significant impact when we are falling back to DCM